### ISSUE TITLE
Importing sometimes results in hydron trying to find a local copy that doesn't exist

### ISSUE THREAD

**Issue by ghost** [bakape/hydron#33]
Not really sure why it does this, am having this problem both on NixOS and on a local build.
This has happened before, and I think I fixed it by having the non-postgres DB import first.

```
/mnt/hdd0/home/okina/Pictures/4chan/Anime/Reaction/1436950787058.jpg: open /var/lib/hydron/.hydron/images/8f/8f6b5b0be19462b26407dfab5879fe66e5fc18e0.jpg: no such file or directory
/mnt/hdd0/home/okina/Pictures/4chan/Anime/Reaction/1436891105637.png: open /var/lib/hydron/.hydron/images/0f/0f5f604be23a26a06e4a815b23de253214c59612.png: no such file or directory
/mnt/hdd0/home/okina/Pictures/4chan/Anime/Reaction/1440632125_1440191487933.png: open /var/lib/hydron/.hydron/images/dd/dd06d0519d179a65d55a90ca4d1437e6364f9d83.png: no such file or directory
/mnt/hdd0/home/okina/Pictures/4chan/Anime/Reaction/1441418695266.jpg: open /var/lib/hydron/.hydron/images/6a/6a40f3922add1f4269be4f6b31305fc6dcba1410.jpg: no such file or directory
/mnt/hdd0/home/okina/Pictures/4chan/Anime/Reaction/1441261665019.gif: open /var/lib/hydron/.hydron/images/ad/ad0ee8024117ea252ed0565406050c738df46853.gif: no such file or directory
/mnt/hdd0/home/okina/Pictures/4chan/Anime/Reaction/1437195725182.jpg: open /var/lib/hydron/.hydron/images/0a/0aa4cc5d76edbe62b3c923ac618826df39123e93.jpg: no such file or directory
/mnt/hdd0/home/okina/Pictures/4chan/Anime/Reaction/1441135913469.jpg: open /var/lib/hydron/.hydron/images/74/740d7ef55d1dc70fad1a9c8b534502413a795ae7.jpg: no such file or directory
```

**Comment by ghost** [comment]
I should mention that neither the images folder nor the hashes are existent.

**Comment by ghost** [comment]
Okay, so I found the lead up to the cause, and possibly the cause itself.
Hydron doesn't seem to properly initialize when it's using postgres right off the bat without initializing "normally" first, and creating certain files in the .hydron folder.

**Comment by ghost** [comment]
Interestingly enough, you don't have to import without postgres, that can be your initial import.
You just **have** to initialize hydron without it the first time.

**Comment by ghost** [comment]
So, from what I can gather, we need to check the boot sequence of hydron and make sure that if we're using postgres, we're going through all the steps we would normally.

**Comment by bakape** [comment]
Okay. 
Idea: To account for dir init being interrupted, generate the needed dir on error with `os.MkdirAll()`. We can actually skip the init step, if all file creation calls use this pattern.

**Comment by ghost** [comment]
On error where?

**Comment by bakape** [comment]
When writing image to disk or initializing DB.

**Comment by bakape** [comment]
Fixed  with 74c094d8ceb28e9ab83469ec095431ec29b08b2a.
